libs/sdk-core/src/error.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::Error;
+use std::fmt;
+
+pub type SdkResult<T, E = NewSdkError> = Result<T, E>;
+
+/// Type of error returned by the SDK
+#[derive(Debug)]
+pub enum NewSdkError {
```

Comment:
We can use just `SdkError` for naming this enum.

Reply:
It compiles, but won't this cause issues with uniffi?

`breez_sdk.udl` would contain:

```udl
[Error]
enum SDKError {
  "Error",
};

[Error]
enum SdkError {
  ...
}
```

Reply:
We should use the new sdk error and import it. We should remove the SDKError from the uniffi_binding.rs

Reply:
Done